### PR TITLE
Adding Golden State Water Company - Claremont OWRS file for 01-01-2018

### PR DESCRIPTION
**Pull request by ArashSeif**

### CODE REVIEW

File: full_utility_rates/California/Golden State Water Company - Claremont - 1181/01-01-2018.owrs [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+        10": 2577.15
+    commodity_charge: flat_rate_commodity*usage_ccf
+    flat_rate_commodity: 3.899
+    fixed_drought_surcharge: 0
+    variable_drought_surcharge: 0
+    fixed_wastewater_charge: 0
+    variable_wastewater_charge: 0
+    bill: service_charge+commodity_charge
+  IRRIGATION:
+    service_charge: 3*number_turn_on
```
Comment (ArashSeif) [inline full_utility_rates/California/Golden State Water Company - Claremont - 1181/01-01-2018.owrs]: @christophertull please review this line.

Comment (patwater) [reply full_utility_rates/California/Golden State Water Company - Claremont - 1181/01-01-2018.owrs]: What is the number_turn_on?

Comment (ArashSeif) [reply full_utility_rates/California/Golden State Water Company - Claremont - 1181/01-01-2018.owrs]: image
Water bill for irrigation customers includes a quantity charge of $0.582 per ccf and a "Turn_on Charge" of $3 for each turn-on.

Comment (patwater) [reply full_utility_rates/California/Golden State Water Company - Claremont - 1181/01-01-2018.owrs]: Please omit the service charge and add a "turn-on_charge"

**Pull request merged**